schema/core.md

```diff
@@ -439,11 +460,7 @@ Multiple funding sources can be expressed as a list:
     "url": "https://nsf.gov/awardsearch/showAward?AWD_ID=2118329",
     "funder": {
        "@type": "Organization",
-       "name": "National Science Foundation",
-       "identifier":[
-        "https://ror.org/021nxhr62",
-        "https://doi.org/10.13039/100000001"
-       ]
+       "name": "National Science Foundation"
```

Comment:
Thinking about this again, I believe it's worthwhile to keep "identifier" for the funding object. This should be optional, but it would enable users to document the official award identifier. This is useful in the following scenarios:

1. We want to enable searching or discovering of resources linked to a specific grant
2. We want to enable discovery of related resources via their connection to a specific grant, i.e. graph connectivity...this might be something we can add to the resource landing page in the future.
3. A resource uses the "url" filed to point to the funders homepage not the project home page, or they use it to point to an unofficial project homepage.

Reply:
After talking with Irene, we made the following decisions.

1. identifier should be removed from "funder"
2. identifier should be added to "funding". This is optional and descriptive text should say something like "a unique code or identifier associated with the funded project, e.g. 1148453.

Reply:
Done.